tests/type-checks.ts

```diff
@@ -86,9 +93,263 @@ async function closureSoICanUseAwait() {
     assert<Has<typeof b, () => Promise<number>>>(true);
     assert<IsAny<typeof b>>(false);
     const subproxy = proxy.c;
-    assert<Has<typeof subproxy, { d: Promise<number> }>>(true);
+    assert<Has<typeof subproxy, Promise<{ d: number }>>>(true);
     assert<IsAny<typeof subproxy>>(false);
     const copy = await proxy.c;
     assert<Has<typeof copy, { d: number }>>(true);
   }
+
+  {
+    Comlink.wrap(new MessageChannel().port1);
+    Comlink.expose({}, new MessageChannel().port2);
+    const connection = new RTCPeerConnection();
+    const channel = connection.createDataChannel("comlink");
```

Comment:
`connection` and `channel` are not used, right? (This scared me :D)

Reply:
Actually, all 4 expressions are unused

Reply:
Yeah, must have been copy pasta from the v3 tests. Removed